packages/babel-parser/src/plugins/typescript/index.js

```diff
@@ -131,6 +131,8 @@ const TSErrors = makeErrorTemplates(
       "A 'set' accessor cannot have rest parameter.",
     SetAccesorCannotHaveReturnType:
       "A 'set' accessor cannot have a return type annotation.",
+    StaticBlockCannotHaveanyModifier:
```

Comment:
```suggestion
    StaticBlockCannotHaveModifier:
```
nit-picking: I prefer shorter error codes.